tests/test_transfer_comp_classifier.py

```diff
@@ -0,0 +1,92 @@
+import unittest
+from scripts import TransferCompClassifier
+from torch import optim
+import torch
+import pathlib
+import numpy as np
+
+
+class TransferCompClassifierTest(unittest.TestCase):
+    """
+    this class tests the TransferCompClassifier
+    This test suite can be ran with:
+        python -m unittest -q tests.TransferCompClassifierTest
+    """
+
+    def setUp(self):
+        self.path_to_saved_model = pathlib.Path(__file__).parent.absolute().joinpath(
+            "models/transweighclassifier_2020-02-26-08_58_55")
```

Comment:
I changed this , it takes a small model that I also added from the test directory now so that we don't have absolute paths